plugins/rbenv/rbenv.plugin.zsh

```diff
@@ -1,60 +1,69 @@
-_homebrew-installed() {
-  type brew &> /dev/null
-}
-
-FOUND_RBENV=0
-rbenvdirs=("$HOME/.rbenv" "/usr/local/rbenv" "/opt/rbenv" "/usr/local/opt/rbenv")
-if _homebrew-installed && rbenv_homebrew_path=$(brew --prefix rbenv 2>/dev/null); then
-    rbenvdirs=($rbenv_homebrew_path "${rbenvdirs[@]}")
-    unset rbenv_homebrew_path
-    if [[ $RBENV_ROOT = '' ]]; then 
-      RBENV_ROOT="$HOME/.rbenv"
+# This plugin loads rbenv into the current shell and provides prompt info via
+# the 'rbenv_prompt_info' function.
+
+FOUND_RBENV=$+commands[rbenv]
+
+if [[ $FOUND_RBENV -ne 1 ]]; then
+    rbenvdirs=("$HOME/.rbenv" "/usr/local/rbenv" "/opt/rbenv" "/usr/local/opt/rbenv")
+    for dir in $rbenvdirs; do
+        if [[ -d $dir/shims ]]; then
+            export PATH="$dir/shims:$dir/bin:$PATH"
+            FOUND_RBENV=1
+            break
+        fi
+    done
+fi
+
+if [[ $FOUND_RBENV -ne 1 ]]; then
+    if (( $+commands[brew] )) && dir=$(brew --prefix rbenv 2>/dev/null); then
+        if [[ -d $dir/shims ]]; then
+            export PATH="$dir/shims:$dir/bin:$PATH"
+            export RBENV_ROOT=$dir
```

Comment:
Are you sure this is necessary? I have `rbenv` installed with Homebrew, but my `RBENV_ROOT` points to `~/.rbenv`.

I think it's not even necessary to define `RBENV_ROOT`. [Installation instructions](https://github.com/rbenv/rbenv#basic-github-checkout) don't even use this env variable.

Reply:
True - it was in the existing plugin. I'll remove it.